test/integration/connector.js

```diff
@@ -110,7 +110,7 @@ describe('Connector', function () {
 	it('should be able to fetch schema', function (next) {
 		connector.fetchSchema(function (err, schema) {
 			should(err).be.not.ok;
-			should(schema).be.an.object;
+			should(schema).be.an.Object;
```

Comment:
These assertions should be functions:
`should(schema).be.an.Object();`
See https://shouldjs.github.io/#assertion-object